### DISCUSSION TITLE
3x week habit > negative score, because of different days

### DISCUSSION THREAD

**Discussion by ghost** [iSoron/uhabits#1197] [Help & FAQ] [answered]
Hi,

i have a habit of doing sports 3x/week. Usually on monday, wednessday and friday. 

![Screenshot_20211125-102136](https://user-images.githubusercontent.com/95022529/143415407-3637ff3f-7b03-47ea-b066-43f0adf05526.png)

![Screenshot_20211125-102141](https://user-images.githubusercontent.com/95022529/143415415-b73b0dd9-7753-4796-9628-e7ee79956e3e.png)

If i do sports on tuesday, thursday and sunday instead (still 3x/week), my score will be slightly negative, why?

![Screenshot_20211125-102230](https://user-images.githubusercontent.com/95022529/143415420-00d6813d-67bf-440e-b0d2-59554a4dcfb9.png)

![Screenshot_20211125-102222](https://user-images.githubusercontent.com/95022529/143415417-1ded11ff-dedc-44d4-84e4-6480cf07671f.png)

**Comment by hiqua** [comment, marked as answer]
See https://github.com/iSoron/uhabits/discussions/689 "How is the score calculated?"
